src/protocollogin.cpp

```diff
@@ -183,6 +184,12 @@ void ProtocolLogin::onRecvFirstMessage(NetworkMessage& msg)
 		return;
 	}
 
+	if (g_config.getBoolean(ConfigManager::MAINTENANCE_MODE)) {
```

Comment:
this one is not needed if you make this line
https://github.com/otland/forgottenserver/blob/master/src/otserv.cpp#L374

depend on config value + give tutors flag `canAlwaysLogin`

the server could start in `GAME_STATE_CLOSED` state, this way it would be identical to `/closeserver` command